torch/_dynamo/variables/distributed.py

```diff
@@ -146,8 +146,11 @@ def call_method(
 
             args = [x.as_python_constant() for x in args]
             kwargs = {k: v.as_python_constant() for k, v in kwargs.items()}
-            method(self.value, *args, **kwargs)
-            return self
+            if name == "__setattr__":
```

Comment:
I'm curious what is the use case of calling ```__setattr__```. This is somehow conflict with L139 where we disallow custom __getattr__. I think (1) if we have custom __setattr__, we should inline the function rather than bake in the value during compilation; (2) we don't have custom __setattr__, actually we should handle set attribute logic like ```x.a = 10``` in ```BuiltinVariable.call_setattr```.

Reply:
Actually @wanchaol I think it was added in this PR - https://github.com/pytorch/pytorch/pull/103146/files#diff-a56c74055de538397fde0bb0e158fa7957f46aba68632995de3b6b519a79d66eR93 maybe we can take it out?

Reply:
I agree to take it out, if there is any issue, we should address them in ```BuiltinVariable.call_setattr```, but I think it should be fine.